pkg/engine/jmespath/functions.go

```
@@ -834,6 +834,14 @@ func jpLabelMatch(arguments []interface{}) (interface{}, error) {
 	return true, nil
 }
 
+func ToBoolean(input string) bool {
```

You probably want to return an error too.

---

Now it's, resolved.